nss-day-cohort-60/rare-client-tenacious-tators
# Lh styling

# Description of Proposed Changes
All author names & posts are clickable & will navigate to their details page. 

---

# Steps to Test

1. Fetch the lh-styling branch and switch to it
2. Npm start
3. Navigate to home page & verify that all author names are links & they navigate to correct detail page. 
4. Make sure when you click on each post, you are navigated to detail page.
